### PR TITLE
Documentation.

### PR DESCRIPTION
**Pull request by aaron-brown**

### CODE REVIEW

**Comment by aaron-brown** [comment]
@budjb: I believe I have addressed all comments.

File: src/docs/guide/factorybeans/hazelcastmapfactorybean.gdoc [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+{code}
+
+The bean can then be used in other managed Spring beans:
+
+{code}
+class MyClass {
+    /**
+     * MyMap
+     *
+    Map<String, Object> myMap
```
Comment (budjb) [inline src/docs/guide/factorybeans/hazelcastmapfactorybean.gdoc]: Autowired

Comment (aaron-brown) [inline src/docs/guide/factorybeans/hazelcastmapfactorybean.gdoc]: According to the example, this map is referenced, as using `@Autowired` causes the validation issues.

Your [earlier comment](https://github.com/budjb/grails-hazelcast/pull/4#discussion_r106280862) acknowledged it was a good note.

Comment (aaron-brown) [inline src/docs/guide/factorybeans/hazelcastmapfactorybean.gdoc]: But, the Javadoc is incorrect.

**Pull request merged**